libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -2015,5 +2015,190 @@
         "value": "Show less",
         "comment": "The button title which will display less description text.",
         "limit": 0
+    },
+    "configurationPanel.drillConfig.learn.more.link": {
+        "value": "Learn more",
+        "comment": "Link with additional info",
+        "limit": 0
+    },
+    "configurationPanel.drillConfig.drillToDashboard.tooltip": {
+        "value": "Open a related dashboard for an expanded view on the selected data point, carrying over relevant filters and context.",
+        "comment": "",
```

Comment:
Please add comments for all messages, these comments will help the agency understand the context of the text.
Thank you so much!

Reply:
done

Reply:
There are some message missing comment: 
- Line: 2134 -> 2152
- Line: 2181

Reply:
I think these messages are quite straightforward and comments are unnecessary